### PR TITLE
Fix gcs links

### PR DESCRIPTION
**Pull request by ocervell**

### CODE REVIEW

File: modules/slo/iam.tf [hunk 1]
```diff
@@ -63,3 +71,10 @@ resource "google_storage_bucket_iam_member" "object-viewer" {
   role   = "roles/storage.objectViewer"
   member = "serviceAccount:${local.sa_email}"
 }
+
+resource "google_storage_bucket_iam_member" "object-viewer-legacy" {
```
Comment (morgante) [inline modules/slo/iam.tf:75]: Why is the legacy role needed?

Comment (ocervell) [reply modules/slo/iam.tf:75]: I noticed errors in the logs that the service account didn't have `storage.buckets.get`:
```
google.api_core.exceptions.Forbidden: 403 GET https://storage.googleapis.com/storage/v1/b/<BUCKET_NAME>?projection=noAcl&prettyPrint=false: <SERVICE_ACCOUNT_NAME>@<PROJECT_ID>.iam.gserviceaccount.com does not have storage.buckets.get access to the Google Cloud Storage bucket.
```
weirdly the `roles/storage.objectViewer` does not include this permission, but the legacy role does.

Comment (ocervell) [reply modules/slo/iam.tf:75]: @morgante should we keep as is or create a custom role with just 1 permission ?

Comment (morgante) [reply modules/slo/iam.tf:75]: That is odd. I think this is fine as-is then.

**Review by morgante — APPROVED** [review]

**Pull request merged**